backend/btrixcloud/operator.py

```diff
@@ -232,6 +232,7 @@ async def sync_crawls(self, data: MCSyncData):
         params["force_restart"] = spec.get("forceRestart")
 
         params["redis_url"] = redis_url
+        params["redis_scale"] = 1 if status.state != "waiting_capacity" else 0
```

Comment:
```suggestion
        params["redis_scale"] = 1 if status.state not in ("starting", "waiting_capacity") else 0
```

If we set it to this, then redis doesn't actually start until crawler pod is creating/running - when crawls are not queued, this does add ~10 seconds to crawl run time, as it first checks that crawler pod is running. When at capacity, does prevent initial burst of redis pods starting up..